kentik_api_library/kentik_api/requests_payload/manual_mitigations_payload.py

```diff
@@ -0,0 +1,26 @@
+from kentik_api.public.manual_mitigation import ManualMitigation
+import json
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass()
+class CreateRequest:
+    ipCidr: str
+    Comment: Optional[str]
+    platformID: str
+    methodID: str
+    minutesBeforeAutoStop: str
+
+
+class CreateResponse:
+    def __init__(self, result: str) -> None:
+        self.response = {"result": result}
+
+    @classmethod
+    def from_json(cls, json_string):
+        dic = json.loads(json_string)
+        return cls(dic["response"]["result"])
+
+    def status(self) -> str:
+        return self.response["status"]
```

Comment:
I dont see where "status" key is being set...? __init__ only sets "result"

Reply:
ack